openthread/README.rst

```diff
@@ -16,182 +16,36 @@ For more information about OpenThread feature sets and how to select them in |NC
-     - ``y``
-     - ``y``
-     - ``n``
-   * - :option:`CONFIG_OPENTHREAD_MTD`
-     - ``n``
-     - ``n``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_IP6_FRAGM`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_JAM_DETECTION`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_JOINER`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_L2_DEBUG`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_L2_LOG_LEVEL`
-     - ``3``
-     - ``3``
-     - ``3``
-   * - :option:`CONFIG_OPENTHREAD_L2_LOG_LEVEL_INF`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_LOG_LEVEL`
-     - ``1``
-     - ``1``
-     - ``1``
-   * - :option:`CONFIG_OPENTHREAD_LOG_LEVEL_CRIT`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_MAC_FILTER`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_MBEDTLS`
-     - ``n``
-     - ``n``
-     - ``n``
-   * - :option:`CONFIG_OPENTHREAD_MTD_NETDIAG`
-     - ``y``
-     - ``n``
-     - ``n``
-   * - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER`
-     - ``y``
-     - ``n``
-     - ``n``
-   * - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_FTD`
-     - ``n``
-     - ``y``
-     - ``n``
-   * - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MTD`
-     - ``n``
-     - ``n``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_NRF_SECURITY`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_PKT_LIST_SIZE`
-     - ``10``
-     - ``10``
-     - ``10``
-   * - :option:`CONFIG_OPENTHREAD_RAW`
-     - ``y``
-     - ``n``
-     - ``n``
-   * - :option:`CONFIG_OPENTHREAD_SHELL`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_SLAAC`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_SNTP_CLIENT`
-     - ``y``
-     - ``y``
-     - ``y``
-   * - :option:`CONFIG_OPENTHREAD_THREAD_VERSION`
-     - ``1.1``
-     - ``1.1``
-     - ``1.1``
-   * - :option:`CONFIG_OPENTHREAD_UDP_FORWARD`
-     - ``y``
-     - ``y``
-     - ``n``
-   * - :option:`CONFIG_OPENTHREAD_MAX_CHILDREN`
-     - ``32``
-     - ``32``
-     - ``32``
-   * - :option:`CONFIG_OPENTHREAD_MAX_IP_ADDR_PER_CHILD`
-     - ``6``
-     - ``6``
-     - ``6``
+   * - NCS tag
+     - SoC
+     - Crypto backend
+     - Thread version
+     - OpenThread commit
+     - GNU Arm Embedded Toolchain version
+     - Nordic features set
+   * - v1.4.0
+     - nRF52840
+     - CC310
+     - 1.1
+     - ``07f430dac6c400abdd358ba1173c0f94b94a5a80``
+     - ``9-2019-q4-major``
+     - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER`
+   * - v1.4.0
+     - nRF52833
+     - nrf_oberon
+     - 1.1
+     - ``07f430dac6c400abdd358ba1173c0f94b94a5a80``
+     - ``9-2019-q4-major``
+     - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER`
```

Comment:
```suggestion
   * - SoC
     - |NCS| tag
     - Crypto backend
     - Thread version
     - OpenThread commit
     - GNU Arm Embedded Toolchain
     - Feature set
   * - nRF52840
     - v1.4.0
     - CC310
     - v1.1
     - ``07f430dac6c400abdd358ba1173c0f94b94a5a80``
     - ``9-2019-q4-major``
     - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER`
   * - nRF52833
     - v1.4.0
     - nrf_oberon
     - v1.1
     - ``07f430dac6c400abdd358ba1173c0f94b94a5a80``
     - ``9-2019-q4-major``
     - :option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER`
```
The first two columns should be inverted, because users are interested in qualifications for their chip (and will switch to a different NCS version if necessary).
I'm wondering if Crypto backend and Thread version also should be inverted - I tried to locate the file mentioned below, and when navigating through the libs, you select v1.1 before you select oberon/cc310. But I'm not sure which one is more relevant to the user - probably the Thread version, and then they use the backend that is available?

Reply:
Agree about SoC as first column. I'll also move the Thread version column before the crypto backend, which is merely informative here.